README.md

```diff
@@ -4,34 +4,13 @@
 [![devDependency Status](https://david-dm.org/ng-bootstrap/core/dev-status.svg?branch=master)](https://david-dm.org/ng-bootstrap/core#info=devDependencies)
 
 Welcome to the Angular 2 version of the [Angular UI Bootstrap](https://github.com/angular-ui/bootstrap) library.
-This library is being built from scratch in TypeScript using the Bootstrap 4 CSS framework.
+This library is being built from scratch in TypeScript using the Bootstrap 4 CSS framework. 
+This project is being created by the [ui-bootstrap team](https://github.com/angular-ui/bootstrap).
 
 As with Bootstrap 4 and Angular 2, this library is a work in progress. Please check out our list of
 [issues](https://github.com/ng-bootstrap/core/issues) to see all the things we are implementing.
 Feel free to make comments there.
 
-This project is being created by the [ui-bootstrap team](https://github.com/angular-ui/bootstrap).
-
-# Quick links
```

Comment:
We don't have that much content so I don't feel like we need those quick links for now.